modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/list/Item.es.js

```diff
@@ -469,7 +469,20 @@ class Item extends PureComponent {
 					<section className="autofit-section">
 						<div className="list-group-title">
 							<span className="text-truncate-inline">
-								{url ? <a href={url}>{title}</a> : title}
+								{viewURL ? (
+									<>
+										<a
+											href={viewURL}
+											rel="noopener noreferrer"
+											target="_blank"
+										>
+											{`${title} `}
+											<ClayIcon symbol="shortcut" />
+										</a>
+									</>
```

Comment:
I don't think we need this wrapped in a fragment since it only has 1 child.

Reply:
Oops... I'll get rid of it!